R/feature_extraction_functions.R

```diff
@@ -69,7 +69,7 @@ time_domain_summary <- function(values, sampling_rate=NA) {
 #' frequency_features = frequency_domain_summary(
 #'   accelerometer_data$x,
 #'   sampling_rate = 100.122,
-#'   npeaks = 4)
+#'   npeaks = 3)
```

Comment:
Why are we changing this to 3?

Reply:
I don't know. You're the one who changed it.

It's also just an example, so maybe it doesn't matter?

Reply:
The npeaks is dependent on the type of signal we are trying to study. If the underlying frequency spectrum is bimodal we would go for npeaks = 2 ideally. 

3 is just an example. Keep it. This just means we will pick the three most dominant peaks.

Reply:
according to your git history you changed it to 3 to reduce the amount of time it takes to run the function (when running travis tests, for example)